Gems/EMotionFX/Code/EMotionFX/Tools/EMotionStudio/Plugins/StandardPlugins/Source/SceneManager/ActorPropertiesWindow.cpp

```diff
@@ -334,6 +334,12 @@ namespace EMStudio
             return;
         }
 
+        // If the names are the same, do not change them.
+        if (std::strcmp(m_actor->GetName(), m_nameEdit->text().toUtf8().data()) == 0)
```

Comment:
`m_nameEdit->text()` is a QString already right? Why not just use its built-in `compare` function?
```suggestion
        if (m_nameEdit->text().compare(QStringView(m_actor->GetName())==0)
```

Reply:
done.